src/test/scala/sbt/errorssummary/BasicConciseReporterSpec.scala

```diff
@@ -91,23 +91,22 @@ class BasicConciseReporterSpec
   it should "strip prefix if told to" in {
     val code                     = """error"""
     val configWithShortenedPaths = defaultConfig.withShortenPaths(true)
-    val expectedText             = "[1] src.scala:1:"
+    val expectedText             = "[E1] src.scala"
 
-    collectMessagesFor(code, configWithShortenedPaths) {
-      (problems, messages) =>
-        problems should have length 1
+    collectMessagesFor(code, configWithShortenedPaths) { (problems, messages) =>
+      problems should have length 1
 
-        messages should have length 2
-        val (_, msg) = messages.head
-        val lines    = msg.split(EOL)
-        lines(0) shouldBe expectedText
+      messages should have length 2
```

Comment:
Btw, it seems that all the tests are checking for `messages` length. Perhaps it's better for only one test (or one set of tests) to do the length checking?